sqlxx/expand.go

```diff
@@ -32,3 +42,52 @@ func (e Expandables) Has(search Expandable) bool {
 	}
 	return false
 }
+
+func (e Expandables) Load(ctx context.Context, c *pop.Connection, m interface{}) error {
+	e.Sort()
+	return e.load(ctx, c, m, 1, e.MaxLevels())
+}
+
+func (e Expandables) load(ctx context.Context, c *pop.Connection, m interface{}, level int, maxLevel int) error {
+	var eg errgroup.Group
+	for i := range e {
+		item := e[i].String()
+		if len(strings.Split(item, ".")) == level {
+			continue
+		}
+
+		eg.Go(func() error {
+			// We need a copy of the connection which is only possible using `WithContext` because
```

Comment:
What does „copy a connection“ mean, exactly?

Reply:
It just copies the struct, not the SQL connection itself :)